bom/openhab-addons/pom.xml

```diff
@@ -675,6 +675,11 @@
       <artifactId>org.openhab.binding.russound</artifactId>
       <version>${project.version}</version>
     </dependency>
+    <dependency>
+      <groupId>org.openhab.addons.bundles</groupId>
+      <artifactId>org.openhab.binding.sinope</artifactId>
```

Comment:
In my alphabet, "si" comes after "sa"...

Reply:
1st grade :)